app/controllers/orders_controller.rb

```diff
@@ -1,24 +1,70 @@
 class OrdersController < ApplicationController
   def cart # One cart (@order) holds information on zero or many OrderItems
-    @order = Order.new(status: "pending")
-    @items = OrderItem.all
+    @order = Order.find_by(status: "pending") # under the assumption there is only one order happening at a time, does this logic work?
```

Comment:
ooh this is interesting. i do wonder if this works. i feel like for our basic implementation it does....do we know that this is only ever going to correspond to one user? (i.e. my brain is thinking, what if multiple people are NOT logged in and using our site and they all add items to their individual carts)

Reply:
Omg yes that totally is something that I didn't think about but would affect this logic. Will need to think about how to find_by with something a little more specific than a "pending" status.